pydoctor/templatewriter/pages/__init__.py

```diff
@@ -196,14 +196,6 @@ def namespace(self, obj: model.Documentable) -> List[Union[Tag, str]]:
         parts.reverse()
         return parts
 
```

Comment:
This will make the Twisted system build pass with a warning instead of failing.
```suggestion
    @renderer
    def deprecated(self, request: object, tag: Tag) -> "Flattenable":
        import warnings
        warnings.warn("Renderer 'CommonPage.deprecated' is deprecated, the twisted's deprecation system is now supported by default.")
```